bin/brew

```diff
@@ -53,7 +53,7 @@ HOMEBREW_LIBRARY="$HOMEBREW_REPOSITORY/Library"
 # Whitelist and copy to HOMEBREW_* all variables previously mentioned in
 # manpage or used elsewhere by Homebrew.
 for VAR in AWS_ACCESS_KEY_ID AWS_SECRET_ACCESS_KEY BINTRAY_USER BINTRAY_KEY \
-           BROWSER EDITOR GIT NO_COLOR PATH VISUAL
+           BROWSER EDITOR GIT NO_COLOR PATH VISUAL AWS_SESSION_TOKEN
```

Comment:
Thanks Mike. I had mentioned above that I tried omitting this and it did not work. I even dropped a puts ENV["HOMEBREW_AWS_SESSION_TOKEN"] into the formula and saw that it was nil when I omitted it, and present when I had this line there. 

Would you be willing to point me to a section of code that makes you think it should work? It would definitely help me understand this homebrew environment stuff a little better.

Reply:
If you set `export HOMEBREW_AWS_SESSION_TOKEN=1` in your environment it will be set as `ENV["HOMEBREW_AWS_SESSION_TOKEN"]` in Homebrew itself.

Reply:
Oh, I see where you're going. There is still a usability piece there that could be avoided if we kept this line. Oftentimes, especially in the case of aws vault, that variable is automatically populated. 

I'm not sure it's as valuable of a change for me as a user if I still have to manipulate environment variables by hand. It's also worth noting that it is a fairly ephemeral value, so it's not something someone can easily toss into a bashrc.

Reply:
> Oftentimes, especially in the case of aws vault, that variable is automatically populated.

Can you elaborate a bit on this?

> I'm not sure it's as valuable of a change for me as a user if I still have to manipulate environment variables by hand.

You could argue conversely; I'm not sure you want to have sensitive AWS information floating around inside the Homebrew process unless set explicitly.